ostelco-ios-client/AppDelegate.swift

```diff
@@ -65,6 +59,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
     
+    func configureAppearance() {
+        if
+            let bundleIndentifier = Bundle.main.bundleIdentifier,
```

Comment:
This `if` code style looks weird

Reply:
ah i need to update the indent on setting `useDevColor`, then it'll be a bit more reasonable. The idea is to put each thing we're checking on its own line and have them all line up. I admit it looks better with `guard` statements, but I've found it to be helpful to be able to parse multiple conditions as I'm reading.